.ci/install-sdk.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+DEST="${GOPATH}/bin/operator-sdk"
+
```

Comment:
Shouldn't it also define the default for `SDK_VERSION`? This way the script would work without the caller knowing the configuration options.

Reply:
If we ever have someone to use this script manually, we can change it. Right now, it's easiest to just use the same target as before: `make install-sdk`

Reply:
fair enough. Defining all the prerequisites helps with portability and readability. But it's just a ci script.